### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by leneherng**
### Changes
- Ballot Seat: Refactor mobile design
- Refactor `Button`, 
- Refactor `WindowProvider`, `useWindow`
- Add `WindowProvider` to all pages that has `StateModal`, `StateDropdown`, `Bar` or `useWindow`
- ElectionCard: fix title
- Data Catalogue: fix analytics all-time to dashboard
- Input: fix search query text colour in dark mode
- Table: fix `sticky-col` when `table-stripe` is used
- Add `box-shadow` style to tailwind
- IPR: Choropleth to monochrome, 
- Choropleth: change outline according to theme
- use divide instead of border-b

### Considerations
- Table: when there's 2 sticky/freeze columns, after turning to next page, issue seems to be `document.getElementById` can't find `cellId`, currently stuck here
- Election Analysis: column width for area is not styled correctly in Mozilla, works in Safari, Chrome-based
- `Button`: is just a wrapper now

### CODE REVIEW

File: apps/app/components/Chart/Table/index.tsx [hunk 1]
```diff
@@ -165,6 +165,7 @@ const Table: FunctionComponent<TableProps> = ({
   );
 
   const calcStickyLeft = (cellId: string) => {
+    // FIXME: document.getElementById returns null when page is turned
```
Comment (irfancoder) [inline apps/app/components/Chart/Table/index.tsx:168]: Yeah, was aware of this issue. Can you create a ticket on trello pls. Thank you

File: apps/app/hooks/useWindow.tsx [hunk 2]
```diff
@@ -42,5 +45,7 @@ export const WindowProvider: FunctionComponent<WindowProviderProps> = ({ childre
     };
   }, []);
 
-  return <WindowContext.Provider value={{ breakpoint, scroll }}>{children}</WindowContext.Provider>;
+  return (
+    <WindowContext.Provider value={{ windowWidth, scroll }}>{children}</WindowContext.Provider>
+  );
```
Comment (irfancoder) [inline apps/app/hooks/useWindow.tsx:50]: Not sure whats the difference here other than the rename.

File: apps/app/pages/dashboard/election-explorer/trivia/[state].tsx [hunk 3]
```diff
@@ -38,38 +43,40 @@ const ElectionTriviaState: Page = ({
...
+  //       },
+  //     },
+  //     {
+  //       params: {
+  //         state: state.key,
+  //       },
+  //       locale: "ms-MY",
+  //     },
+  //   ]);
+  // });
```
Comment (irfancoder) [inline apps/app/pages/dashboard/election-explorer/trivia/[state].tsx:79]: delete the commented code

File: apps/app/pages/dashboard/fire-and-rescue/[state].tsx [hunk 4]
```diff
@@ -35,38 +40,41 @@ const FireandRescueState: Page = ({
...
+  //       },
+  //     },
+  //     {
+  //       params: {
+  //         state: state.key,
+  //       },
+  //       locale: "ms-MY",
+  //     },
+  //   ]);
+  // });
```
Comment (irfancoder) [inline apps/app/pages/dashboard/fire-and-rescue/[state].tsx:74]: delete the commented code

**Review by irfancoder — CHANGES_REQUESTED** [review]

File: apps/app/hooks/useWindow.tsx (hunk 2 above)
Comment (leneherng) [reply apps/app/hooks/useWindow.tsx:50]: Ah yes its just a rename, I forgot I decided to use the original implementation

**Review by irfancoder — APPROVED** [review]

**Pull request merged**